sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -433,6 +433,61 @@ class JDBCSuite extends QueryTest
     assert(ids(2) === 3)
   }
 
+  test("SPARK-34843: columnPartition should generate a correct stride size") {
+
```

Comment:
nit: unnecessary blank.

Reply:
This is a stylistic habit I have from my workplace--we always put a LF after opening a brace to make the code less "stuffy." I've removed the extra LF to keep it consistent with the rest of the code base; will be reflected in my next push.